meilisearch-lib/src/document_formats.rs

```diff
@@ -136,16 +143,23 @@ pub fn read_json(input: impl Read, writer: impl Write + Seek) -> Result<usize> {
 
     let content: ArrayOrSingleObject = serde_json::from_reader(reader)
         .map_err(Error::Json)
-        .map_err(|e| (PayloadType::Json, e))?;
+        .map_err(|e| {
+            println!("Błąd o taki: {:#?}", e);
+            (PayloadType::Json, e)
+        })?;
 
+    println!("content o taki: {:#?}", content);
     for object in content.inner.map_right(|o| vec![o]).into_inner() {
+        println!("{:#?}", object);
         builder
             .append_json_object(&object)
             .map_err(Into::into)
             .map_err(DocumentFormatError::Internal)?;
     }
 
     let count = builder.documents_count();
+    println!("{count}");
```

Comment:
Would you please remove the `println`s?

Reply:
Yes, obviously. 
Very sorry about that 😅